packages/cli/src/watcher.js

```diff
@@ -118,7 +109,20 @@ function subscribe(
     compileDirectory(
       config,
       resp.files.filter((file) => file.exists).map((file) => file.name),
-      resp.files.filter((file) => !file.exists).map((file) => file.name),
-    );
+      resp.files
+        .filter(
+          (file) =>
+            // don't trigger recompile when the cli deletes the compiled modules folder
+            !file.exists && !file.name.startsWith('stylex_compiled_modules'),
+        )
+        .map((file) => file.name),
+    )
+      .then(() => {
+        clearInputModuleDir(config);
+      })
+      .catch((transformError) => {
```

Comment:
NVM.

Reply:
Yeah, throwing the error only when it's not in watch mode. Otherwise just console.erroring it